website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -461,6 +461,7 @@ variables is shown in the table below.
 | `db_admin_username`                                   | string        | Username for RDS database administrator authentication.                                                 | "postgres" |
 | `default_multipart_chunksize_mb`                      | number        | The default maximum size of chunks to use when copying. Can be overridden by collection config.         | 250 |
 | `metadata_queue_message_retention_time_seconds`       | number        | Number of seconds the metadata-queue fifo SQS retains a message.                                        | 777600 |
+| `db_name`                                             | string        | The name of the Orca database within the RDS cluster.                                                   | disaster_recovery_PREFIX |
```

Comment:
See comment on name being PREFIX_name instead of name_PREFIX

Reply:
Fixed